suites/suite_held_suarez_1994.xml

```diff
@@ -10,6 +10,14 @@
     <scheme>sima_state_diagnostics</scheme>
   </group>
   <group name="physics_after_coupler">
+    <!-- <scheme>physics_dme_adjust</scheme> -->
+
+    <!-- MPAS and SE specific scaling of temperature for enforcing energy consistency:
+         First, calculate the scaling based off cp_or_cv_dycore (from cam_thermo_water_update)
+         Then, perform the temperature and temperature tendency scaling -->
+    <scheme>check_energy_scaling</scheme>
+    <scheme>dycore_energy_consistency_adjust</scheme>
+
```

Comment:
Sorry, we don't want this here (Held-Suarez is considered "dry physics"), but instead in `suite_tj2016.xml`, or here:

https://github.com/ESCOMP/atmospheric_physics/blob/main/suites/suite_tj2016.xml

Reply:
My apologies, updated! I also added comments to the scheme .xml where available based on the previous review comment.